.gitlab-ci/packet.yml

```diff
@@ -115,6 +115,11 @@ packet_ubuntu18-cilium-sep:
   extends: .packet
   when: manual
 
+packet_ubuntu18-cilium-svc-proxy:
+  stage: deploy-special
+  extends: .packet
+  when: manual
```

Comment:
Why is this `manual`? is that job supposed to work or not?

I did a manual trigger to check: https://gitlab.com/kargo-ci/kubernetes-sigs-kubespray/-/jobs/614699759

Reply:
As the default for kube_proxy_remove is not working properly in master (fixed by #5554). The kube-proxy will not be removed and ciliIum in strict mode do not expect that, so this test fails.

I put this test in manual mode because the test for a basic cilium install is also triggered manually but I can change if you want

Reply:
let's address first #5554 and then enable the job to test the remove kube-proxy use case

Reply:
#5554 has been merged now :)